index.js

```diff
@@ -0,0 +1,107 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+const bcrypt = require('bcryptjs');
+
+
+
+const knexConfig = require('./knexfile');
+
+const db = knex(knexConfig.development);
+
+const session = require('express-session');
+const knexSessionStore = require('connect-session-knex')(session);
+
+const server = express();
+
+
+
+const sessionConfig = {
```

Comment:
great job setting up the session configurations and adding the comments to the side to help more easily show what it's suppose to be doing